packages/vm/src/dynamic_link.rs

```diff
@@ -14,10 +14,9 @@ use wasmer_types::ImportIndex;
 
 use cosmwasm_std::{from_slice, Addr};
 
-const MAX_REGIONS_LENGTH: usize = 100_000;
-
 // The length of the address is 63 characters for strings and 65 characters with "" for []byte. Thus, 65<64*2 is used.
 const MAX_ADDRESS_LENGTH: usize = 64 * 2;
+const MAX_INTERFACE_REGIONS_LENGTH: usize = 8 * 1024 * 1024;
```

Comment:
Please, write the comment about `MAX_INTERFACE_REGIONS_LENGTH`.
And why has this value increased so much?